docs/debugging.rst

```diff
@@ -3,68 +3,142 @@ Debugging
 
 This section includes documentation on how to debug Flatpak apps.
 
-Running debugging tools
------------------------
+Debug packages
```

Comment:
This whole section is replaced by a single command:

`flatpak install --include-sdk --include-debug $FLATPAK_ID`

Reply:
Very useful, thanks.